internal/utils/files.go

```diff
@@ -39,3 +42,32 @@ func MoveFile(sourcePath, destPath string) error {
 	}
 	return nil
 }
+
+func MoveDirectory(source, dest string) error {
+	err := copy.Copy(source, dest)
```

Comment:
Will `Copy` fail if `dest` already exists? I prefer it will.

Reply:
It seems that it *will* fail (see https://github.com/otiai10/copy/blob/745c20085b466efba6d5d30071f02b8303d5ba30/copy.go#L118) and also in the case that a destination file exists.

Reply:
Looking at Go's documentation I see that it will not fail but return `nil`: https://golang.org/pkg/os/#MkdirAll.

I also see that after creating the directories it uses `os.Create` and according it's doc "If the file already exists, it is truncated". This is a problem since it might delete user configuration.

I suggest checking if the file exists. if it is a directory, we can continue to copy if it doesn't exist or it does but it's empty. That way we will be on the safe side.

Reply:
That's a good point.

I've done that now. I also added a test for checking whether ``dest`` exists, too.